virt/virt-4-8-release-notes.adoc

```diff
@@ -170,3 +170,5 @@ data:
 ----
 $ ovirt-aaa-jdbc-tool user unlock admin
 ----
+
+* A "No registered model" warning message appears if you customize a virtual machine in the *Customize virtual machine* section when creating a VM using the VM wizard. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1977658[*BZ#1977658*])
```

Comment:
I think it's important to note that this message is not harmful and can be ignored (as per the bug).

I also might reorder this slightly so that the context is provided upfront.

Finally, the BZ that you link should be the engineering bug, not the doc bug (when there are two separate bugs). In this case, it's https://bugzilla.redhat.com/show_bug.cgi?id=1974689, which is the bug that BZ1977658 was cloned from.

I'd suggest something like this:

```
* If you create a VM by using the wizard, a "No registered model" warning appears when you customize the VM. You can safely ignore this warning. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1974689[*BZ#1974689*])
```